Resources/Prototypes/_Carpmosia/Decals/bricktile_half_dark.yml

```
@@ -0,0 +1,83 @@
+﻿- type: decal
+  id: BrickTileDarkHalfCornerNe
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: dark_half_corner_ne
+
+- type: decal
+  id: BrickTileDarkHalfCornerSe
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: dark_half_corner_se
+
+- type: decal
+  id: BrickTileDarkHalfCornerNw
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: dark_half_corner_nw
+
+- type: decal
+  id: BrickTileDarkHalfCornerSw
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: dark_half_corner_sw
+
+- type: decal
+  id: BrickTileDarkHalfInnerNe
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: dark_half_inner_ne
+
+- type: decal
+  id: BrickTileDarkHalfInnerSe
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: dark_half_inner_se
+
+- type: decal
+  id: BrickTileDarkHalfInnerNw
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: dark_half_inner_nw
+
+- type: decal
+  id: BrickTileDarkHalfInnerSw
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: dark_half_inner_sw
+
+- type: decal
+  id: BrickTileDarkHalfN
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: dark_half_n
+
+- type: decal
+  id: BrickTileDarkHalfS
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: dark_half_s
+
+- type: decal
+  id: BrickTileDarkHalfE
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: dark_half_e
+
+- type: decal
+  id: BrickTileDarkHalfW
+  tags: ["station", "markings"]
+  sprite:
+    sprite: _Carpmosia/Decals/brickhalftile.rsi
+    state: dark_half_w    
```

```suggestion
    state: dark_half_w

```
You should keep newlines at the end of all files